src/sentry/integrations/github/issues.py

```diff
@@ -51,12 +51,12 @@ def get_generic_issue_body(self, event: GroupEvent) -> str:
     def get_group_description(self, group: Group, event: Event | GroupEvent, **kwargs: Any) -> str:
         output = self.get_group_link(group, **kwargs)
 
-        if isinstance(event, GroupEvent) and event.occurrence is not None:
-            body = self.get_generic_issue_body(event)
-            output.extend([body])
-        elif group.issue_category == GroupCategory.PERFORMANCE:
+        if group.issue_category == GroupCategory.PERFORMANCE:
             body = self.get_performance_issue_body(event)
             output.extend([body])
+        elif isinstance(event, GroupEvent) and event.occurrence is not None:
+            body = self.get_generic_issue_body(event)
+            output.extend([body])
```

Comment:
I don't think we should switch this, we want to prioritise using the occurrence when available here

Reply:
do you think I can continue using `self.get_performance_issue_body` here?

Reply:
Yeah we can do that if you prefer, not a huge deal